attributes/default.rb

```diff
@@ -2130,3 +2130,4 @@
                                            'topologyEdgeTopic' => 'edges', 'metricTopic' => 'metrics', 'eventFilter' => '', 'alarmFilter' => '', 'forward.metrics' => false,
                                            'nodeRefreshTimeoutMs' => 300000, 'alarmSyncIntervalMs' => 300000, 'suppressIncrementalAlarms' => true, 'kafkaSendQueueCapacity' => 1000,
                                            'startAlarmSyncWithCleanState' => false}
+default['opennms']['kafka']['producer'] ['client'] = {}
```

Comment:
I think there's an extra space here? and maybe add a short comment above explaining what the attribute does